### PR TITLE
Update README & Refactor (Move create schema API from create.go to schema.go)

### PR DESCRIPTION
**Pull request by imakhlin**
* Update README 
* Add API for creating default schema programmatically, not just for testing
* Move create DB business logic from create command to schema.go
* Update tests accordingly

### CODE REVIEW

File: pkg/tsdb/schema/schema.go [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+	"github.com/pkg/errors"
+	"github.com/v3io/v3io-tsdb/pkg/aggregate"
+	"github.com/v3io/v3io-tsdb/pkg/config"
+	"strings"
+)
+
+const (
+	SchemaVersion                 = 0
+	DefaultStorageClass           = "local"
+	DefaultIngestionRate          = ""
```
Comment (talIguaz) [inline pkg/tsdb/schema/schema.go]: It is not used. either delete it for now, or calculate the chunk&partition interval from the rate.

File: pkg/partmgr/partmgr_test.go [hunk 2]
```diff
@@ -72,14 +72,18 @@ func TestCreateNewPartition(tst *testing.T) {
 }
 
 func getPartitionManager(tst *testing.T) *PartitionManager {
-	schema := testutils.CreateSchema(tst, "*")
+	schm, err := schema.NewSchema([]string{"*"})
```
Comment (talIguaz) [inline pkg/partmgr/partmgr_test.go]: You could just left it as is, and let the testutil.CreateSchema deal with errors and test fails.

File: pkg/tsdbctl/create.go [hunk 3]
```diff
@@ -115,13 +103,13 @@ func (cc *createCommandeer) create() error {
 	defaultRollup := config.Rollup{
```
Comment (talIguaz) [inline pkg/tsdbctl/create.go]: Why don't just use the `NewShema` here also?

File: pkg/tsdbctl/create.go [hunk 4]
```diff
@@ -138,20 +126,20 @@ func (cc *createCommandeer) create() error {
...
 		AggregatorsGranularity: cc.rollupInterval,
-		StorageClass:           defaultStorageClass,
+		StorageClass:           schema.DefaultStorageClass,
 		SampleRetention:        cc.sampleRetention,
 		ChunckerInterval:       tableSchema.ChunckerInterval,
 		PartitionerInterval:    tableSchema.PartitionerInterval,
 	}
 
-	schema := config.Schema{
+	schm := config.Schema{
```
Comment (talIguaz) [inline pkg/tsdbctl/create.go]: why don't just use the `NewSchema` here also?

File: pkg/tsdb/schema/schema.go (hunk 1 above)
Comment (imakhlin) [reply pkg/tsdb/schema/schema.go]: user MUST provide `IngestionRate` therefore NO DEFAULT value

File: pkg/tsdb/schema/schema.go [hunk 5]
```diff
@@ -1,40 +1,66 @@
...
 	DefaultAggregationGranularity = "1h"
 	DefaultShardingBuckets        = 8
 	DefaultLayerRetentionTime     = "1y"
 	DefaultSampleRetentionTime    = 0
-	DefaultPartitionInterval      = "48h"
-	DefaultChunkInterval          = "1h"
 )
 
-func NewSchema(aggregates []string) (*config.Schema, error) {
+func NewSchema(sampleRate, aggregatorGranularity, aggregatesList string, minChunkSize, maxChunkSize, maxSampleSize, maxPartitionSize, sampleRetention, shardingBuckets int) (*config.Schema, error) {
```
Comment (talIguaz) [inline pkg/tsdb/schema/schema.go]: I wouldn't expose the parameters for the rate calculation to the user. the way to change them is by config if you want. (`minChunkSize, maxChunkSize, maxSampleSize, maxPartitionSize`)

**Pull request merged**